common.mk

```diff
@@ -99,6 +99,7 @@ COMMONOBJS    = array.$(OBJEXT) \
 		enumerator.$(OBJEXT) \
 		error.$(OBJEXT) \
 		eval.$(OBJEXT) \
+		event_profiling.$(OBJEXT) \
```

Comment:
Adding a header needs new dependencies.

Reply:
Sorry I don't understand well, could you please explain? Thank you.

Reply:
The dependencies on event_profiling.[ch] files will be needed in "AUTOGENERATED DEPENDENCIES" at the bottom of common.mk file.

Reply:
https://github.com/ruby/ruby/pull/4673/checks?check_run_id=3155279692#step:10:1

Reply:
Shall I add it manually? Or it can be **auto generated** as it said? I don't know this part.

Reply:
Oh I understood. Thank you.